app/src/main/java/net/squanchy/schedule/view/ScheduleDayPageView.kt

```diff
@@ -52,3 +57,19 @@ class ScheduleDayPageView @JvmOverloads constructor(
         override fun areContentsTheSame(oldItemPosition: Int, newItemPosition: Int) = oldEvents[oldItemPosition] == newEvents[newItemPosition]
     }
 }
+
+private class SnappingLinearLayoutManager(context: Context) : LinearLayoutManager(context) {
```

Comment:
I'd put this and `TopSnappedSmoothScroller` in their own file

Reply:
Why is that? This class is (so far) only used here, so to me it makes sense to have it close to the place where it's used. If we find more uses for it (or it grows in terms of LOC) we can think about moving it to its separate file.

Reply:
Mostly in terms of mental encapsulation. It's a nit for me too, anyway, but if it's a de facto internal class maybe it should be an internal class?

Reply:
Sure, it can be an internal class, but it must not be marked as `inner`. I placed it outside so to declutter the main class, but we can compromise on having it inside :)